src/main/scala/com/linkedin/feathr/offline/source/dataloader/jdbc/SQLType.scala

```diff
@@ -0,0 +1,30 @@
+package com.linkedin.feathr.offline.source.dataloader.jdbc
+
+import org.apache.spark.sql.{DataFrame, SparkSession}
+
+sealed trait SQLType
+
+object SQLType {
```

Comment:
class doc

Reply:
the naming is confusing with Spark SQL Type. Make it more specific, like SqlDatabaseType